### PR TITLE
CB-10760: Document plugin element

### PR DESCRIPTION
**Pull request by omefire**

### CODE REVIEW

**Comment by omefire** [comment]
@rakatyal , could you please help review this ?

File: www/docs/en/dev/config_ref/index.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ platform. See [Customize icons topic](images.html) for more information.
     <allow-intent href="sms:*" />
 
 ## engine
-   Specifies what platform to restore during a prepare. For more details, see the [Platforms & Plugins Version Management](../platform_plugin_versioning_ref/index.html).
+   Specifies what platform to restore during a prepare. For more details, see the [Platforms & Plugins Version Management](../platform_plugin_versioning_ref/index.html#platform-versioning).
```
Comment (rakatyal) [inline www/docs/en/dev/config_ref/index.md]: Can we rename the link to the updated title: "Manage versions and platforms"?

File: www/docs/en/dev/config_ref/index.md [hunk 2]
```diff
@@ -199,6 +199,18 @@ platform. See [Customize icons topic](images.html) for more information.
...
 
     <engine name="android" spec="^4.0.0" />
+    
+## plugin
+   Specifies what plugin to restore during a prepare. For more details, see the [Platforms & Plugins Version Management](../platform_plugin_versioning_ref/index.html#plugin-versioning).
+
+   Attributes(type) | Description
+   ----------------- | ------------
+   name(string) | *Required* <br/> Name of the plugin to be restored
+   spec(string) | *Required* <br/> Version of the plugin to be restored
```
Comment (nikhilkh) [inline www/docs/en/dev/config_ref/index.md]: Spec is not the `version` - correct? I describe it in detail here: https://cordova.apache.org/docs/en/dev/cordova-cli/index.html#plugin-spec

File: www/docs/en/dev/config_ref/index.md (hunk 1 above)
Comment (nikhilkh) [inline www/docs/en/dev/config_ref/index.md]: The TOC tile should really be renamed "Manage platforms and plugins"

Comment (omefire) [inline www/docs/en/dev/config_ref/index.md]: "Manage platforms and plugins" doesn't really convey the intent behind the info included in that link. It brings to my mind actions like: add/rm/update platforms/plugins.

I think the original "Platforms & Plugins Versioning" was much clearer, versioning being the keyword here.

**Comment by riknoll** [comment]
:+1:

**Pull request merged**